Classes/Configuration/AutomaticConfigurator.php

```diff
@@ -83,7 +83,7 @@ protected function addLanguages(array &$configuration) {
 		if ($this->hasStaticInfoTables) {
 			$languages = $this->databaseConnection->exec_SELECTgetRows('t1.uid AS uid,t2.lg_iso_2 AS lg_iso_2', 'sys_language t1, static_languages t2', 't2.uid=t1.static_lang_isocode AND t1.hidden=0');
 		} else {
-			$languages = $this->databaseConnection->exec_SELECTgetRows('t1.uid AS uid,t1.uid AS lg_iso_2', 'sys_language t1', 't1.hidden=0');
+			$languages = $this->databaseConnection->exec_SELECTgetRows('t1.uid AS uid,t1.language_isocode AS lg_iso_2', 'sys_language t1', 't1.hidden=0');
```

Comment:
`language_isocode` is not present in TYPO3 6.2, which this ext claims compatibility for.

Core-wise we have available in
6.2: static_lang_isocode, which points to a static_languages records if present
7.6: language_isocode holds the iso-code directly (always), static_lang_isocode is there for backwardscompat when SIT is installed